config/dev/onebox.yml

```diff
@@ -55,12 +48,8 @@ EDXAPP_MYSQL_USER: '%%MYSQL_USER%%'
 EDXAPP_MYSQL_PASSWORD: '%%MYSQL_PASSWORD%%'
 
 # Configure DNS for site and email links
-EDXAPP_LMS_SITE_NAME: "%%LMS_URL%%"
+EDXAPP_SITE_NAME: "%%LMS_URL%%"
```

Comment:
why would you change the config name?

Reply:
this assignment either happens for us in edx-configuration or the original variable name is unused. i'll check which case it was